src/pages/Login.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory } from 'react-router';
+import Input from '../helpers/Input';
+import Button from '../helpers/Button';
+import logo from '../images/mustachef.svg';
+
+export default function Login() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [disable, setDisable] = useState(true);
+  const [validEmail, setValidEmail] = useState(false);
+  const [validPassword, setValidPassword] = useState(false);
+
+  const minPasswordLength = 6;
+  const history = useHistory();
+
+  const login = () => {
+    localStorage.setItem('mealsToken', 1);
+    localStorage.setItem('cocktailsToken', 1);
+    const emailInfo = JSON.stringify({ email });
+    localStorage.setItem('user', emailInfo);
+    history.push('/comidas');
+  };
+
+  const validateFields = () => {
+    if (validEmail && validPassword) {
+      return setDisable(false);
+    }
+    return setDisable(true);
+  };
+
+  const validateEmail = (value) => {
+    setEmail(value);
+    const regex = /\S+@\S+\.\S+/;
+    setValidEmail(regex.test(value));
+  };
+
+  const validatePassword = (value) => {
+    setPassword(value);
+    if (value.length > minPasswordLength) {
+      setValidPassword(true);
+    }
```

Comment:
adicionar para senha voltar a inválida
```suggestion
    }else { setValidPassword(false); }
```